### PR TITLE
Fixes #21858: Command seq not found in crontab on AIX

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/21858

### CODE REVIEW

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Review by amousset — CHANGES_REQUESTED** [review]
This does not work as expected

File: techniques/system/common/1.0/cron-setup.cf [hunk 1]
```diff
@@ -46,7 +47,11 @@ bundle agent setup_cronjob
         edit_defaults => noempty_backup,
         edit_line => delete_lines_matching("0,5,10,15,20,25,30,35,40,45,50,55 \* \* \* \* /opt/rudder/bin/rudder agent check -q >> /var/log/rudder/agent-check/check.log 2>&1");
 
- }
+      # Cleanup crontab with seq error, with the tag
+      "/var/spool/cron/crontabs/root"
+        edit_defaults => noempty_backup,
+        edit_line => delete_lines_matching("sh: seq:.*# RUDDER CRON");
```
Comment (amousset) [inline techniques/system/common/1.0/cron-setup.cf:53]:
```suggestion
```

Comment (amousset) [reply techniques/system/common/1.0/cron-setup.cf:53]: Replace 

```
edit_line     => insert_lines("${cron_prefix} /opt/rudder/bin/rudder agent check -q >> /var/log/rudder/agent-check/check.log 2>&1 # RUDDER CRON"),
```

by 

```
edit_line     => converge("# RUDDER CRON", "${cron_prefix} /opt/rudder/bin/rudder agent check -q >> /var/log/rudder/agent-check/check.log 2>&1 # RUDDER CRON"),
```

Maintains a clean file

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**